### PR TITLE
docs: create /security endpoint

### PR DESCRIPTION
**Pull request by philips**
Create an easy to remember and locate URL for security disclosure process.
This URL will need to be placed in lots of templates and tools so make it

This is the new disclosure process as discussed here:
https://github.com/kubernetes/kubernetes/issues/35462

This relies on a doc to be merged into
docs/devel/security-release-process.md but this doc can be reviewed in
parallel.

You can find a draft of the content of that doc on #35462

cc @jessfraz @erictune @alex-mohr @eparis

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1819)

### CODE REVIEW

File: security/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Any vulnerability information shared with Product Security Team stays within Kubernetes project and will not be disseminated to other projects unless it is necessary to get the issue fixed.
+
+As the security issue moves from triage, to identified fix, to release planning we will keep the reporter updated.
+
+## Public Disclosure Timing
+
+A public disclosure date is negotiated by the Kubernetes product security team and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (esp. if it's already publicly known) to a few weeks. As a basic default, we expect report date to disclosure date to be on the order of 7 days. The Kubernetes product security team holds the final say when setting a disclosure date.
```
Comment (liggitt) [inline security/index.md]: s/esp./especially/

**Comment by jessfraz** [comment]
nice! LGTM minus the nit above

**Comment by philips** [comment]
Nit addressed. We need to pause until I get the dev docs merged.

**Comment by philips** [comment]
Dev docs for the security process are up: https://github.com/kubernetes/kubernetes/pull/37836

**Pull request merged**